app/templates/app/add_contracts.html

```diff
@@ -5,64 +5,51 @@
 {% load staticfiles %}
 {% load render_table from django_tables2 %}
 {% load i18n %}
-{% block title %} Upfronts {% endblock title %}
-{% block content %}
-<html>
-    <head>
-        <link rel="stylesheet" href="../../static/css/contract_table.css">
-        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.0/css/all.css">
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
-        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
-        <script src="https://unpkg.com/gijgo@1.9.13/js/gijgo.min.js" type="text/javascript"></script>
-        <link href="https://unpkg.com/gijgo@1.9.13/css/gijgo.min.css" rel="stylesheet" type="text/css" />
-        <title>Add Installment</title>
+{% block head %}
+    {{ block.super }}
+    <link rel="stylesheet" href="../../static/css/contract_table.css">
+    <script src="https://unpkg.com/gijgo@1.9.13/js/gijgo.min.js" type="text/javascript"></script>
+    <link href="https://unpkg.com/gijgo@1.9.13/css/gijgo.min.css" rel="stylesheet" type="text/css" />
+    <script src="{% static 'js/add_contracts.js' %}"></script>
+{% endblock %}
 
-    </head>
-    <body>
-        <div class='p-3'>
-            {% if message %}
-                <br>
-                <div class="alert alert-danger" role="alert">
-                    {{message}}
+{% block title %} Add contract {% endblock title %}
+
+{% block content %}
+    <div class='p-3'>
+        {% if message %}
+            <br>
+            <div class="alert alert-danger" role="alert">
+                {{message}}
+            </div>
+        {% endif %}
+        <div>
+            <div class="m-2">
+                <h1>Search cases</h1>
+            </div>
+            <form>
+                <div class="form-row">
+                    <div class="col-2">
+                        <input class="form-control" type="text" placeholder="Enter case numbers" name="case_numbers">
+                    </div>
+                    <div class="col-2">
+                        <input id="datepicker_from" placeholder="From" name="case_date_from">
+                    </div>
+                    <div class="col-2">
+                        <input id="datepicker_to" placeholder="To" name="case_date_to">
+                    </div>
+                    <div class="col">
+                        <button type="submit" class="btn btn-primary datep"><i class="fab fa-sistrix"></i></button>
+                    </div>
                 </div>
+            </form>
+            {% if table %}
+                <br>
```

Comment:
Pls. Do this with css. The break tag was created to create an space between text lines https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br